### PR TITLE
Add e2e test for delegates monitor screen - Closes #4101

### LINKED ISSUE #4101: Add e2e tests for delegates monitor screen
### Description

As a part of #4065 we'd like to introduce e2e test coverage for the delegates monitor screen. The e2e tests should test the following user journeys.

### Acceptance criteria
- Delegates overview
   - Total blocks should be correct.
   - Total transactions should be correct.
- Forging details
   - Blocks forged should change every 10 seconds.
   - Minutes passed should be correct.
   - Next forgers should display 6 delegates which match the first members of the inside round list.
- Inside round
   - Should display 103 delegates. 101 of them being active.
   - Should display forging status correctly.
   - Should allow to sort by forging time.
   - Should allow to filter by username.
   - Should allow to add to watch list.
- Outside round
   - Should display standby and ineligible delegates.
   - Should allow to load more.
   - Should allow to sort by status.
   - Should allow to filter by username.
   - Should allow to add to watch list.
- Sanctioned
   - Should display punished and banned delegates.
   - Should allow to load more.
   - Should allow to filter by username.
   - Should allow to add to watch list.
- Watch list
   - Should display delegates who are added to watch list.
   - Should allow removing items from the watch list.
- Latest votes
   - Should display vote txs including first few delegates.
   - Should allow to load more.
   - Each tx should be clickable. 
- If Lisk Desktop throws an unwanted error that fails the e2e test in a random basis, it should be resolved.

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #4101 

### How was it solved?

Writing e2e tests against the following scenarios:

- [x] Should properly display delegates overview
- [x] Should properly display forging details
- [x] Inside round should be properly functioning
- [x] Outside round should be properly functioning
- [x] Sanctioned delegates should be functioning properly
- [x] Watch list should be functioning properly
- [x] Latest votes should be functioning properly

### How was it tested?

Jenkins

### CODE REVIEW

File: test/cypress/features/delegates.feature [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Feature: delegate
+     Scenario: I should be on the delegates page
+          Given Network is set to devnet
+          And I am on delegates page
+          And I wait 5 seconds
+
+     Scenario: Should properly display delegates overview
+          Then totalBlocks count should have value greater than 0
+          And transactions count should have value greater than 0
```
Comment (ikem-legend) [inline test/cypress/features/delegates.feature:9]: @eniolam1000752 Please format the file spacing similar to other test files

File: test/cypress/features/delegates/delegates.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import { ss } from '../../../constants';
+
+const getDelegateNameFromRow = (ele) => ele.find('span:first-child > div > div:last-child > div:last-child > p:first-child').text();
+
+/* eslint-disable */
+Then(/(\w+) count should have value greater than (\d+)/, function(displayElementClassName, displayContent){
```
Comment (ikem-legend) [inline test/cypress/features/delegates/delegates.js:6]: @eniolam1000752 Please move `eslint-disable` to the top of the file and import `Then` and `When` statements from cucumber

File: test/cypress/features/delegates/delegates.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        const delegateName = getDelegateNameFromRow(ele)
+        if(delegateName === this.watchedDelegate){
+            expect(ele.find('span:first-child > div:first-child > div:first-child > span:first-child ~ div > main')).contain('Remove from watched')
+        }
+      })
+  }); 
+
+  Then(/^there should be more than (\d+) delegates in table$/, function (number) {
+    cy.get(ss.delegateRow).should('have.length.gte', number);
+  });
```
Comment (ikem-legend) [inline test/cypress/features/delegates/delegates.js:130]: @eniolam1000752 I fail to see where this is used

File: src/components/screens/monitor/delegates/overview/forgingDetails.js [hunk 4]
```diff
@@ -125,7 +125,8 @@ const ForgingDetails = ({
                   <h6>{t('Blocks forged')}</h6>
                   <ProgressBar forgedInRound={forgedInRound} theme={theme} />
                   <p className={styles.blue}>
-                    {`${forgedInRound} / `}
+                    <span className="blocksForged">{`${forgedInRound} `}</span>
```
Comment (ikem-legend) [inline src/components/screens/monitor/delegates/overview/forgingDetails.js:128]: @eniolam1000752 There seems to be some unnecessary space after `forgedInRound` in the object literal

File: test/cypress/features/delegates/delegates.js (hunk 2 above)
Comment (eniolam1000752) [reply test/cypress/features/delegates/delegates.js:6]: even with that, there is this lint error A function with a name starting with an uppercase letter should only be used as a constructor. which I think can only be resolved if Then and When are declared as constructors

File: src/components/screens/monitor/delegates/overview/forgingDetails.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (ikem-legend) [reply src/components/screens/monitor/delegates/overview/forgingDetails.js:128]: @eniolam1000752 That's fine then

File: test/cypress/features/delegates/delegates.js [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  cy.get(`${ss.delegateRow}`).should('have.length.within', 0, 103);
+});
+
+When(/^I watch a delegate$/, function () {
+  cy.get(`${ss.delegateRow}`).eq(0).then((ele) => {
+    const watchToggleBtn = ele.find('span:first-child > div:first-child > div:first-child > span:first-child');
+    this.watchedDelegate = getDelegateNameFromRow(ele);
+    watchToggleBtn.trigger('click');
+  });
+});
```
Comment (ikem-legend) [inline test/cypress/features/delegates/delegates.js:110]: @eniolam1000752 Please add an extra line space between blocks

**Review by ikem-legend — APPROVED** [review]

**Review by gadjacobs — APPROVED** [review]

**Pull request merged**